rero_ils/modules/items/views/api_views.py

```diff
@@ -572,7 +573,11 @@ def claim_notification_preview(item_pid):
     if not record.is_issue:
         abort(400, 'Item isn\'t an issue')
 
-    issue_data = record.dumps(dumper=ClaimIssueNotificationDumper())
+    try:
+        issue_data = record.dumps(dumper=ClaimIssueNotificationDumper())
+    except (TypeError, MissingDataException) as exp:
+        abort(500, str(exp))
```

Comment:
Can we return an other status than `500`? Is that means that the request should not be done for this kind of items? Or is it a mistake in the data?

Reply:
I don't found any other HTTP status code : It's an error from server (not from client sending data) --> so 5xx family error. According https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_server_errors, 500 seems the best possible option.